app/services/oneoffs/fix_participant_declaration_user.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Oneoffs
+  class FixParticipantDeclarationUser
+    def call
+      ActiveRecord::Base.transaction do
+        incorrect_participant_declarations.in_batches.each_record do |declaration|
+          if declaration_has_transferred_user?(declaration)
+            declaration.update!(user_id: declaration.participant_profile.user.id)
```

Comment:
is this true for ECF users as well? Or do we need to do any induction record lookups?

Reply:
Mook has confirmed we dont need IR lookups